### PR TITLE
New README

### PR DESCRIPTION
**Pull request by jjasghar**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,60 @@
...
 [![Code Climate](https://img.shields.io/codeclimate/github/test-kitchen/kitchen-openstack.svg)][codeclimate]
 [![Coverage Status](https://img.shields.io/coveralls/test-kitchen/kitchen-openstack.svg)][coveralls]
 [![Dependency Status](https://img.shields.io/gemnasium/test-kitchen/kitchen-openstack.svg)][gemnasium]
 
-[gem]: https://rubygems.org/gems/kitchen-openstack
-[travis]: https://travis-ci.org/test-kitchen/kitchen-openstack
-[codeclimate]: https://codeclimate.com/github/test-kitchen/kitchen-openstack
-[coveralls]: https://coveralls.io/r/test-kitchen/kitchen-openstack
-[gemnasium]: https://gemnasium.com/test-kitchen/kitchen-openstack
+A [Test Kitchen][kitchenci] Driver for [OpenStack][openstack_web].
```
Comment (andrewelizondo) [inline README.md]: you've got some markdown errors here, I'm assuming you want these to be links somewhere.

Comment (jjasghar) [inline README.md]: Good catch, thanks.

Comment (andrewelizondo) [inline README.md]: almost, you made the kitchenci link labeled kitchen_ci, so you've got a mismatch

**Comment by andrewelizondo** [comment]
lgtm :shipit:

**Pull request merged**